src/effects/builtin/amplify/AmplifyView.qml

```diff
@@ -31,76 +30,124 @@ EffectBase {
 
     Column {
 
-        anchors.fill: parent
+        height: implicitHeight
+        width: parent.width
         spacing: 16
 
-        Row {
-            anchors.horizontalCenter: parent.horizontalCenter
+        Column {
 
-            spacing: 4
+            height: implicitHeight
+            width: parent.width
+            spacing: 8
 
             StyledTextLabel {
-                anchors.verticalCenter: parent.verticalCenter
-
-                text: qsTrc("effects/amplify", "Amplification (dB):")
+                text: qsTrc("effects/amplify", "Amplification")
             }
 
-            RealInputField {
-                anchors.verticalCenter: parent.verticalCenter
-                width: 80
+            Row {
+
+                width: parent.width - spacing
+                spacing: 16
 
-                min: amplify.ampMin
-                max: amplify.ampMax
-                decimals: 4
+                StyledSlider {
+                    id: slider
 
-                currentValue: amplify.amp
-                onCurrentValueChanged: {
-                    amplify.amp = currentValue
+                    anchors.verticalCenter: parent.verticalCenter
+
+                    width: parent.width * .65
+
+                    value: amplify.amp
+                    to: amplify.ampMax
+                    from: amplify.ampMin
+                    stepSize: 0.01
+
+                    onMoved: {
+                        if (value !== amplify.amp) {
+                            amplify.amp = value
+                        }
+                    }
                 }
-            }
-        }
 
-        StyledSlider {
-            id: slider
-            width: parent.width
+                IncrementalPropertyControl {
```

Comment:
ahh, I'm glad we replaced RealInputField with IncrementalPropertyControl
but please check https://github.com/audacity/audacity/pull/8050 on regression

Reply:
@Eism yes, 8050 is back indeed, and it should be fixed for `IncrementalPropertyControl` as well, I'll make the PR today